Api/Controllers/ApiController.cs

```diff
@@ -52,12 +52,11 @@ public string LoginUser(DtoLogin dtoLogin)
         // client must have a role to use this http request
         // if you are not authenticated then the response will be a 401 because you don't have access to it
         [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
-        [HttpGet("User/GetClaim")]
+        [HttpPost("User/GetClaim")]
         public List<ModelClaimHealth> GetUserClaims(DtoGetUserClaims dtoGetUserClaims)
         {
-            ModelClaimHealth modelClaimHealth = new ModelClaimHealth();
-            modelClaimHealth.UserId = dtoGetUserClaims.UserId;
-            return _iBusinessGetUserClaim.GetUserClaims(modelClaimHealth);
+            int userId = dtoGetUserClaims.UserId;
```

Comment:
you should use the following code to get the user email from there claims in JWT to get the currently logged in users data by email since email is set to unique in the database. the following string will give you a string with the users email that you can pass down to the repo layer and use in your query to get the users claims. You can then use the HttpGet instead of post since there will be no body and you wont need the dto for userid
string userEmail = ($"{this.User.FindFirst(ClaimTypes.Email)!.Value}");

Reply:
Everything is fixed now :D